### PR TITLE
Expand documentation page about statistical issues

### PR DESCRIPTION
**Pull request by teddygroves**
Just filling out the documentation website a little - this page explains Maud's approach from a high-level statistical point of view.

### CODE REVIEW

**Comment by teddygroves** [comment]
One question that arose for me was why we use measurements of enzyme concentrations and unbalanced metabolite concentrations to directly set priors rather than using the same kind of measurement model we use for flux and balanced metabolite concentration measurements. I wrote that this was for simplicity but I might be forgetting another reason - if anyone remembers please say!

File: docs/theory/statistics.rst [hunk 1]
```diff
@@ -2,9 +2,84 @@
...
+  state)
+- concentrations of enzymes
+
+The prior distributions for formation energies come from an upstream analysis
+involving in vitro measurements and inference about the composition of each
+compound. Kinetic parameter priors are inferred from online databases using the
+'respectful modelling' paradigm. Priors for concentrations of unbalanced
+metabolites and enzyme concentrations are determined respectively by
+metabolomics and proteomics measurements where these are available and by
+reasonable physical limits otherwise.
```
Comment (NicholasCowie) [inline docs/theory/statistics.rst]: We've changed it now haven't we? Now the priors for unbalanced metabolites and enzyme concentrations are just physiologically reasonable approximations, which could be tightened if needed.

File: docs/theory/statistics.rst [hunk 2]
```diff
@@ -2,9 +2,84 @@
...
+
+The numbers that Maud treats as measurements are metabolomics data relating to
+balanced metabolites and fluxomics data meausuring the steady-state rates of
+reactions. Balanced metabolite concentration measurements are treated as draws
```
Comment (NicholasCowie) [inline docs/theory/statistics.rst]: Now enzymes and unbalanced metabolites are also treated as measurements

File: docs/theory/statistics.rst [hunk 3]
```diff
@@ -2,9 +2,84 @@
...
+from a log-normal distribution with known standard deviation parameter and mean
+parameter :math:`\log(true\ concentration)`. Flux measurements are similarly
```
Comment (NicholasCowie) [inline docs/theory/statistics.rst]: And the **True** concentration is that simulated by the model, or updated from the HMC algorithm (unbalanced metabolites and enzyme concentrations)?

File: docs/theory/statistics.rst [hunk 4]
```diff
@@ -2,9 +2,84 @@
...
+main difficulties are (a) specifying how parameter values determine true steady
+state concentrations and fluxes and (b) making the resulting problem
+computationally tractable despite the need to solve a large system of
+non-linear equations.
+
+Statistical assumptions
+=======================
+
+It is somewhat unusual to use metabolomics data to specify prior distributions
+directly rather than adding an extra measurement model. This choice was made
```
Comment (NicholasCowie) [inline docs/theory/statistics.rst]: In house, this is something that we can consider though. Given a number of technical replicates we can fit these measurements as a calibration model parallel and in series to our model. Michael Betancourt's "falling in love with principled modelling" has a nice example of this.

**Comment by teddygroves** [comment]
Abandoning as a lot of the material in here is quite out of date now - I'll copy and paste the salvageable bits into a future documentation.

**Pull request closed without merging**